### DISCUSSION TITLE
Retroarch Metal SHA256 Mismatch

### DISCUSSION THREAD

**Discussion by fatihturan** [Homebrew/discussions#4046] [Casks] [answered]
Homebrew showing SHA256 mismatch error for the `retroarch-metal` cask package:

```
Casks
=====
retroarch-metal (1.13.0) != 1.14.0
==> Casks with 'auto_updates true' or 'version :latest' will not be upgraded; pa
==> Upgrading 1 outdated package:
retroarch-metal 1.13.0 -> 1.14.0
==> Upgrading retroarch-metal
==> Downloading https://buildbot.libretro.com/stable/1.14.0/apple/osx/universal/
######################################################################## 100.0%
==> Purging files for version 1.14.0 of Cask retroarch-metal
Error: retroarch-metal: SHA256 mismatch
Expected: 89162d663b45cdde938cdaf042abe2a23af40dae5fc3ac33f448b3e711c0d4a3
  Actual: fe69da6fad230895c476c9d0f81c02cdbd2c64355dfe8f8c7921ec0183102bbd
    File: /Users/fatihturan/Library/Caches/Homebrew/downloads/63decfc7e318338b6891fca376c5681ec5cd80c83ce074eceefbb6401a273414--RetroArch_Metal.dmg
To retry an incomplete download, remove the file above.
```

**Comment by carlocab** [comment]
https://docs.brew.sh/Common-Issues#cask---checksum-does-not-match

**Comment by fatihturan** [comment]
@carlocab I already followed the instructions for the fixing this error:

>First, check if the problem was with your download. Delete the downloaded file (its location will be pointed out in the error message) and try again.

I deleted the downloaded file before the install but no luck.

> 1. Go to the vendor’s website (brew home <cask_name>).
> 2. Find out what the latest version is. It may be expressed in the URL used to download it.

![image](https://user-images.githubusercontent.com/26871/207639361-4023c099-bd38-42dd-8c69-e693343064eb.png)

> 3. Take a look at the cask’s version (brew info <cask_name>) and verify it is indeed outdated. If so, it will need to be updated.

![image](https://user-images.githubusercontent.com/26871/207639457-93c2f5bf-b1e1-4449-8264-ad258b8e6236.png)

**Reply by carlocab** [reply]
Have you tried submitting a fix?

> Help us by [submitting a fix](https://github.com/Homebrew/homebrew-cask/blob/HEAD/CONTRIBUTING.md#updating-a-cask). If you get stumped, [open an issue](https://github.com/Homebrew/homebrew-cask/issues/new?template=01_bug_report.md) explaining your steps so far and where you’re having trouble.

**Comment by miccal** [comment, marked as answer]
https://github.com/Homebrew/homebrew-cask/pull/137799

**Comment by RJPercival** [comment]
It looks like this has occurred again, but with v1.15.0 of `retroarch`. Is there a bug causing this to reoccur?

**Reply by RJPercival** [reply]
In fact, there's a history of bad SHA256s: https://github.com/orgs/Homebrew/discussions/3189
